src/OpenTelemetry/Metrics/AggregatorStore.cs

```diff
@@ -67,7 +67,6 @@ internal MetricAgg[] MapToMetrics(string[] seqKey, object[] seqVal)
 
                 if (this.instrument.GetType().Name.Contains("Counter"))
                 {
-                    metricpairs.Add(new MetricAgg(timeperiod, new SumMetricAggregator(name, dt, tags, false, true)));
                     metricpairs.Add(new MetricAgg(timeperiod, new SumMetricAggregator(name, dt, tags, true, true)));
```

Comment:
removed one aggregator to see a single output. obviously, these sections would be changing a lot in subsequent PRs/pending PRs.